spyder/plugins/editor.py

```diff
@@ -995,6 +995,13 @@ def get_plugin_actions(self):
         showindentguides_action = create_action(self, _("Show indent guides."),
                                                toggled=self.toggle_show_indent_guides)
         showindentguides_action.setChecked(CONF.get('editor', 'indent_guides'))
+
+        show_classfunc_dropdown_action = create_action(
+                self, _("Show class function dropdown."),

```

Comment:
Please change this to

    Show class and methods dropdown

Reply:
In the preferences says: `Show Class/Function Dropdown`, should I change it too?

Reply:
I wasn't aware of that. Then let's leave it (in both places) as

    Show selector for classes and functions